easybuild/easyconfigs/p/Python/Python-3.6.4-intel-2018a.eb

```diff
@@ -29,6 +29,8 @@ dependencies = [
 
 osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
 
+optimized = False
```

Comment:
I think something like this definitely requires a motivating comment what issues it causes.

Reply:
I think this PR can be dropped in favour of [this easyblock one](https://github.com/easybuilders/easybuild-easyblocks/pull/2228).  I only experienced problems with `intel/2018a` but from what I gather, I was not alone.  I also agree that I should have added a motivating comment, next time...